package.json

```diff
@@ -13,23 +13,23 @@
   },
   "private": false,
   "dependencies": {
-    "@angular/animations": "^5.1.0",
-    "@angular/cdk": "^5.0.1",
-    "@angular/common": "^5.1.0",
-    "@angular/compiler": "^5.1.0",
-    "@angular/core": "^5.1.0",
-    "@angular/forms": "^5.1.0",
-    "@angular/http": "^5.1.0",
-    "@angular/material": "^5.0.1",
-    "@angular/platform-browser": "^5.1.0",
-    "@angular/platform-browser-dynamic": "^5.1.0",
-    "@angular/platform-server": "^5.1.0",
-    "@angular/router": "^5.1.0",
-    "core-js": "^2.4.1",
+    "@angular/animations": "^5.2.2",
```

Comment:
Wouldn't something like `>=5.2.2` be better. Please see https://github.com/dhlab-basel/Salsah/issues/198#issuecomment-371665942

Reply:
I don't think it would make sense because this repo implements the interface `DateApapter` (https://github.com/angular/material2/blob/dfe8091d959f1bb54cc33a8d942cbe7ea4e3fe56/src/lib/core/datetime/date-adapter.ts#L21) which is defined in `@angular/material`. If this interface changes in a future version, that would break the current implementation of this repo.

Reply:
ok, it's important that you and @kilchenmann coordinate when Salsah updates its package versions.

Reply:
Yes, absolutely!